eventiq_asb/broker.py

```diff
@@ -53,14 +47,21 @@ def __init__(
         self._publisher: ServiceBusSender | None = None
         self._renever: AutoLockRenewer | None = None
         self._receivers: dict[int, ServiceBusReceiver] = {}
+        self._publisher_lock = anyio.Lock()
+        self._sb_manager: ServiceBusManager | None = None
```

Comment:
z tego co widze ten ServiceBusManager wymaga paczki aiohttp, więc proponowałbym to zrobić tak, żeby zaimplementować to jako Middleware. Może rozważyć czy renewera też by się nie dało tak zrobić, żeby nie stackować tych prywatnych/opcjonalnych atrybutów i mieć możliwość włączenia i wyłączenia tego.

Reply:
Manager przeniesiony do middleware

Reply:
Renewer przeniesiony do middleware